src/netcdf_io/calc_analysis.fd/inc2anl.f90

```diff
@@ -196,12 +197,22 @@ subroutine add_increment(fcstvar, incvar)
           call read_vardata(fcstncfile, fcstvar, work3d_bg, nslice=k, slicedim=3)
           ! get increment
           incncfile = open_dataset(incr_file, paropen=.true.)
-          call read_vardata(incncfile, trim(incvar)//"_inc", work3d_inc, nslice=k, slicedim=3)
-          ! add increment to background
-          do j=1,nlat
-             jj=nlat+1-j ! increment is S->N, history files are N->S
-             work3d_bg(:,j,1) = work3d_bg(:,j,1) + work3d_inc(:,jj)
-          end do
+          ! JEDI-derived increments have a time dimension, so read to appropriate array
+          if ( jedi ) then
```

Comment:
This is great. I suspect when we move from the AuxGrid in FV3-JEDI to the generic Gaussian interpolation/write in OOPS, this boolean will be even more important.